docs/components/Intro.js

```diff
@@ -558,7 +558,13 @@ function Explained() {
       </div>
 
       <div className="landing-card-section">
-        <h2 className="h2-landing">Devtools</h2>
+        <a className="header-link" href="/devtools/">
```

Comment:
We don't have a /devtools page anymore.

We probably need dedicated headers for tctl and Web UI